gobblin-core/src/main/java/org/apache/gobblin/writer/PartitionedDataWriter.java

```diff
@@ -130,7 +130,8 @@ public PartitionedDataWriter(DataWriterBuilder<S, D> builder, final State state)
       this.state.setProp(WRITER_LATEST_SCHEMA, builder.getSchema());
     }
     long cacheExpiryInterval = this.state.getPropAsLong(PARTITIONED_WRITER_CACHE_TTL_SECONDS, DEFAULT_PARTITIONED_WRITER_CACHE_TTL_SECONDS);
-    this.writeTimeoutInterval = cacheExpiryInterval / 3;
+    // Increase the timeout value to make it less sensitive to HDFS slow writer
+    this.writeTimeoutInterval = cacheExpiryInterval / 3 * 2;
```

Comment:
Do we want to derive this from Cache TTL value or can this be an independent config? The reason being the right value for write timeout may vary depending on the destination being written to. And if the config exceeds the cache ttl, we can bound the value to the TTL.